permiso/build.gradle

```diff
@@ -45,7 +45,7 @@ android {
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
     testCompile 'junit:junit:4.12'
-    compile 'com.android.support:appcompat-v7:23.3.0'
+    compile 'com.android.support:appcompat-v7:26.+'
```

Comment:
Can you specify a concrete version number here? i.e. `26.0.2` or whatever it is.